module.nix

```diff
@@ -91,6 +91,11 @@ in {
           description = "Extra lines to add to pull request body";
           default = "";
         };
+        cooldown = mkOption {
+          type = int;
+          description = "Cooldown duration between updating pull requests";
```

Comment:
Should probably specify that it's in milliseconds

Reply:
Fixed